packages/client/test/integration/test-node/test/compatibility/compatibility.spec.ts

```diff
@@ -0,0 +1,245 @@
+/**
+ * This file is for Compatibility Matrix tests, which assigns specific
+ * Allure context tags.
+ */
+
+import * as allure from 'allure-vitest'
+import { pipe } from 'fp-ts/function'
+import { beforeEach, describe, expect, TaskContext, test } from 'vitest'
+import { clientFactory, setupPeerTestsLifecycle } from '../util'
+import { datamodel, sugar } from '@iroha2/data-model'
+import { CLIENT_CONFIG } from '@iroha2/test-configuration'
+import { Seq } from 'immutable'
+
+setupPeerTestsLifecycle()
+
+beforeEach(async (ctx) => {
+  await allure.owner(ctx, 'dulger')
+  await allure.label(ctx, 'permission', 'no_permission_required')
+  await allure.label(ctx, 'sdk', 'Java Script')
```

Comment:
Consider removing these attributes from `beforeEach` and putting them into `assignAllureAttributes` - so that they are all defined in a single place.

Reply:
ready